PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -163,7 +163,7 @@ void Buffer::updateTimeStamp()
 				std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
 				std::string msg = converter.to_bytes(_fullPathName);
 				char buf[1024];
-				sprintf(buf, "  in updateTimeStamp(): timeStampLive (%u/%u) < _timeStamp (%u/%u)", timeStampLive.dwLowDateTime, timeStampLive.dwHighDateTime, _timeStamp.dwLowDateTime, _timeStamp.dwHighDateTime);

```

Comment:
Use `%lu` since `DWORD` is `long unsigned int`. https://docs.microsoft.com/en-us/cpp/c-runtime-library/format-specification-syntax-printf-and-wprintf-functions?view=msvc-170#size-prefixes-for-printf-and-wprintf-format-type-specifiers
https://en.cppreference.com/w/cpp/io/c/fprintf